From fc0f3e37918f727956df096fc301a0e62926c1c0 Mon Sep 17 00:00:00 2001
From: Evan Huus <eapache@gmail.com>
Date: Sun, 22 Sep 2013 16:42:21 +0000
Subject: [PATCH 13/14] Manual backport of r52142 and r52144 to fix collision
 between gluster defines and system header dirent.h

svn path=/trunk-1.10/; revision=52173
---
 epan/dissectors/packet-gluster.h   | 18 +++++++++---------
 epan/dissectors/packet-glusterfs.c | 18 +++++++++---------
 2 files changed, 18 insertions(+), 18 deletions(-)

diff --git a/epan/dissectors/packet-gluster.h b/epan/dissectors/packet-gluster.h
index e3c43d1..fc69e6c 100644
--- a/epan/dissectors/packet-gluster.h
+++ b/epan/dissectors/packet-gluster.h
@@ -356,15 +356,15 @@ enum gf_fop_procnum {
 
 /* dir-entry types from libglusterfs/src/compat.h */
 enum gluster_entry_types {
-	DT_UNKNOWN = 0,
-	DT_FIFO = 1,
-	DT_CHR = 2,
-	DT_DIR = 4,
-	DT_BLK = 6,
-	DT_REG = 8,
-	DT_LNK = 10,
-	DT_SOCK = 12,
-	DT_WHT = 14
+	GLUSTER_DT_UNKNOWN = 0,
+	GLUSTER_DT_FIFO = 1,
+	GLUSTER_DT_CHR = 2,
+	GLUSTER_DT_DIR = 4,
+	GLUSTER_DT_BLK = 6,
+	GLUSTER_DT_REG = 8,
+	GLUSTER_DT_LNK = 10,
+	GLUSTER_DT_SOCK = 12,
+	GLUSTER_DT_WHT = 14
 };
 
 
diff --git a/epan/dissectors/packet-glusterfs.c b/epan/dissectors/packet-glusterfs.c
index 56f9093..29ff0dc 100644
--- a/epan/dissectors/packet-glusterfs.c
+++ b/epan/dissectors/packet-glusterfs.c
@@ -2002,15 +2002,15 @@ static value_string_ext glusterfs3_1_fop_proc_vals_ext = VALUE_STRING_EXT_INIT(g
 
 /* dir-entry types */
 static const value_string glusterfs_entry_type_names[] = {
-	{ DT_UNKNOWN, "DT_UNKNOWN" },
-	{ DT_FIFO,    "DT_FIFO" },
-	{ DT_CHR,     "DT_CHR" },
-	{ DT_DIR,     "DT_DIR" },
-	{ DT_BLK,     "DT_BLK" },
-	{ DT_REG,     "DT_REG" },
-	{ DT_LNK,     "DT_LNK" },
-	{ DT_SOCK,    "DT_SOCK" },
-	{ DT_WHT,     "DT_WHT" },
+	{ GLUSTER_DT_UNKNOWN, "DT_UNKNOWN" },
+	{ GLUSTER_DT_FIFO,    "DT_FIFO" },
+	{ GLUSTER_DT_CHR,     "DT_CHR" },
+	{ GLUSTER_DT_DIR,     "DT_DIR" },
+	{ GLUSTER_DT_BLK,     "DT_BLK" },
+	{ GLUSTER_DT_REG,     "DT_REG" },
+	{ GLUSTER_DT_LNK,     "DT_LNK" },
+	{ GLUSTER_DT_SOCK,    "DT_SOCK" },
+	{ GLUSTER_DT_WHT,     "DT_WHT" },
 	{ 0, NULL }
 };
 static value_string_ext glusterfs_entry_type_names_ext = VALUE_STRING_EXT_INIT(glusterfs_entry_type_names);
-- 
1.8.3.1

